### PR TITLE
Unittests

### PR DESCRIPTION
**Pull request by simonThiele**
Good day!

Here is the next bunch of tests. Geometry is not fully tests because it shares the same tests as Object3D.
Critical opinions are welcome as always :)

Thx and good evening!

### CODE REVIEW

File: test/unit/core/Raycaster.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+test( "intersectObjects", function() {
+	var raycaster = getRaycaster();
+	var objectsToCheck = getObjectsToCheck();
+
+	ok ( raycaster.intersectObjects( objectsToCheck ).length === 1,
+		"no recursive search should lead to one hit" );
+
+	ok ( raycaster.intersectObjects( objectsToCheck, true ).length === 3,
+		"recursive search should lead to two hits" );
```
Comment (leitzler) [inline test/unit/core/Raycaster.js]: Just a note, shouldn't the text say "... lead to three hits" ? :) (also at line 33)

Comment (simonThiele) [inline test/unit/core/Raycaster.js]: Sry, I wrote the test with two sphere first and then forgot to change the comment. Thx!

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
While you're at it. Would you be up for removing the lodash dependency?

https://github.com/mrdoob/three.js/blob/dev/test/unit/unittests_three.html#L10

**Comment by simonThiele** [comment]
Image of SP
Shure #7652

**Comment by mrdoob** [comment]
Haha!

**Pull request merged**